src/osd/PG.cc

```diff
@@ -6593,6 +6601,30 @@ void PG::_delete_some(ObjectStore::Transaction *t)
   }
 }
 
+// Compute pending backfill data
+static int64_t pending_backfill_kb(CephContext *cct, int64_t bf_bytes, int64_t local_bytes)
+{
+    lgeneric_dout(cct, 20) << __func__ << " Adjust local usage " << (local_bytes >> 10)
+		               << " primary usage " << (bf_bytes >> 10) << dendl;
+    return std::max((int64_t)0, (bf_bytes >> 10) - (local_bytes >> 10));
+}
+
+int PG::pg_stat_adjust(osd_stat_t &new_stat)
```

Comment:
can we pass a pointer instead of ref to communicate that the argument is modified?